CHANGELOG.md

```diff
@@ -4,10 +4,13 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
  
-## [3.0.0] - 2020-02-27
+## [3.0.1] - 2020-08-27
```

Comment:
3.0.0 entry should remain as is. It was released on 2020-02-27. 3.0.1 should go on top with just the trends improvements.

We need to merge master into develop to make sure the 3.0.0 changes are there as well (if we haven't done so already).

Reply:
I'll fix this. I think the second change, i.e. trends regions performance improvement, ended up under 3.0.0 by mistake, because it was a recent change.

Reply:
It should be resolved now.